server/src/database/database.service.ts

```
@@ -19,33 +40,42 @@ export class DatabaseService implements OnModuleInit {
+        '⚠️ Skipping database connection test (database client unavailable)',
       );
       return;
     }
 
     try {
       // Test the connection by querying the conversions table
-      const result = await this.db.select().from(schema.conversions).limit(1);
+      const result = await this.db
+        .select()
+        .from(schema.conversions)
+        .limit(1);
```

ESLint

[error] Replace `⏎········.select()⏎········.from(schema.conversions)⏎········` with `.select().from(schema.conversions)`

(prettier/prettier)